iotedgedev/utility.py

```diff
@@ -93,7 +89,7 @@ def get_file_contents(file, expandvars=False):
                 return content
 
     def decode(self, val):
-        return val.decode("utf-8").strip()
+        return val.decode().strip()
```

Comment:
do we no longer need to pass the encoding as a parameter?

Reply:
no. python 3 defaults to utf-8